app/templates/src/main/resources/config/liquibase/changelog/_initial_schema.xml

```diff
@@ -173,6 +173,36 @@
                                  constraintName="fk_evt_pers_audit_evt_data"
                                  referencedColumnNames="event_id"
                                  referencedTableName="jhi_persistent_audit_event"/>
+    <% if (enableSocialSignIn) { %>
+    <!-- Social -->
+    <createTable tableName="jhi_UserConnection">
```

Comment:
I use the default one: https://github.com/spring-projects/spring-social/tree/master/spring-social-core/src/main/java/org/springframework/social/connect/jdbc. I am working to make compatible with the naming. I rewrite the class `JdbcConnectionRepository.java` and `JdbcUsersConnectionRepository.java` using spring Data JPA.